data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```diff
@@ -6,58 +6,129 @@
       "_source": {
         "enabled": true
```

Comment:
can you also add eventName which is missing: https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto#L226

We have done some analysis on logs mapping here: https://github.com/opensearch-project/opensearch-catalog/issues/217

Reply:
@KarstenSchnitter It looks like we need to bump up the version number to 3 (at least for spans). Otherwise new template mappings are not being sent to the OpenSearch server.

Reply:
Have we added this?

Reply:
Also, this could look much nicer, if I could use component templates. That way, I could use the same dynamic mappings on all three templates. Is there a way to do that with Data Prepper?